lib/nrf_modem_lib/netif/netif.c

```diff
@@ -452,3 +450,14 @@ int lte_connectivity_options_get(struct conn_mgr_conn_binding *const if_conn, in
 
 	return 0;
 }
+
+/* Bind L2 connectity APIs.
```

Comment:
nit:
```suggestion
/* Bind L2 connectivity APIs.
```

Reply:
In fact I would go so far as to suggest 

`/* Bind conn_mgr APIs */`

`L2 connectivity` is I think a mistaken reference to an old name

Reply:
ok, will update